contrib/babelfishpg_tsql/src/pl_handler.c

```diff
@@ -3432,6 +3473,241 @@ bbf_ProcessUtility(PlannedStmt *pstmt,
 				}
 				break;
 			}
+		case T_GrantStmt:
+			{
+				GrantStmt *grant = (GrantStmt *) parsetree;
+				char	   *dbname = get_cur_db_name();
+				const char *current_user = GetUserNameFromId(GetUserId(), false);
+				/* If object is table. */
```

Comment:
ACL_TARGET_OBJECT is for named objects specified in GRANT statement. Please update the comment to reflect the same.